src/main/guidance/chatWidget/ChatWidget.tsx

```
@@ -40,6 +40,13 @@ import ChatWidgetTitle from './ChatWidgetTitle';
 import useChatGuidanceCommunication from './useChatGuidanceCommunication';
 import ConfirmationDialog from '@/core/ui/dialogs/ConfirmationDialog';
 
+// Remove the default props from the Widget component as it's throwing a warning
+// we should revise using this library or fork and fix it properly - fixing it in source is
+// rather easy
+if (Widget.hasOwnProperty('defaultProps')) {
```

Biome (1.9.4)

[error] Do not access Object.prototype method 'hasOwnProperty' from target object.

It's recommended using Object.hasOwn() instead of using Object.hasOwnProperty().
See MDN web docs for more details.

(lint/suspicious/noPrototypeBuiltins)